packages/flutter/lib/src/cupertino/segmented_control.dart

```diff
@@ -0,0 +1,146 @@
+            new Container(
+                decoration:
+                new BoxDecoration(color: CupertinoColors.activeBlue),
+                constraints:
+                new BoxConstraints(maxWidth: 2.0, minWidth: 2.0)),
+            index + 1));
+      }
+    }
+
+    return Container(
+        alignment: Alignment.center,
+        color: const Color(0xFFFFFFFF),
+        child: new Padding(
+            padding: EdgeInsets.symmetric(horizontal: 5.0),
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: <Widget>[
+                new SharedState<int>(
+                  value: sharedState,
+                  valueChanged: (int state) {
+                    setState(() {
+                      sharedState = state;
+                    });
+                  },
+                  child: new ClipRRect(
+                    borderRadius: new BorderRadius.all(Radius.circular(3.0)),
+                    child: new Container(
+                      decoration: new BoxDecoration(
+                          color: CupertinoColors.white,
+                          border: new Border.all(
+                              color: CupertinoColors.activeBlue, width: 2.0)),
+                      child: new Row(
+                          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                          children: buttons),
+                    ),
+                  ),
+                ),
+              ],
+            )));
+  }
+}
+
+class SegmentedControlButton extends StatefulWidget {
+  SegmentedControlButton(this.selectState, this.identifier);
+
+  Widget selectState;
+  final int identifier;
+  bool isDivider;
+
+  _SegmentedControlButtonState createState() => _SegmentedControlButtonState();
+}
+
+class _SegmentedControlButtonState extends State<SegmentedControlButton> {
+  bool isPressed = false;
+
+  _SegmentedControlButtonState();
+
+  @override
+  Widget build(BuildContext context) {
+    final int toggleState = SharedState.getSharedState<int>(context);
+    final bool isSelected = toggleState == widget.identifier;
+    final TextStyle textStyle = DefaultTextStyle.of(context).style.copyWith(
+      color:
+      isSelected ? CupertinoColors.white : CupertinoColors.activeBlue,
+    );
+
+    if (widget.selectState.runtimeType == Text) {
+      widget.selectState =
+      new Text((widget.selectState as Text).data, style: textStyle);
+    } else if(widget.selectState.runtimeType == Icon) {
+      widget.selectState =
+      new Icon((widget.selectState as Icon).icon, color: (isSelected ? CupertinoColors.white : CupertinoColors.activeBlue));
+    }
+    Widget button = new Container(
+        constraints: new BoxConstraints(maxHeight: 28.0),
+        decoration: new BoxDecoration(
+          color: (widget.identifier % 2 != 0)
+              ? CupertinoColors.activeBlue
+              : (isPressed
+              ? Color.fromRGBO(0, 122, 255, 0.2)
```

Comment:
Where did this color come from?  Did you measure it somewhere?

Reply:
Yes, we found this color specified in the UI Design Guides Adobe XD file.

Reply:
OK, great.  Can you make this into `static const` constant, and instead of using fromRGBO, use `const Color(0x33007aff)`.  That will avoid some math in the production code.

Reply:
Done